Expensify/App
# Trim Room title on the report details page

### Details


### Fixed Issues

$ https://github.com/Expensify/App/issues/7181

### Tests | QA Steps
1. Go to staging.new.expensify.com
2. log in expensifail account
3. Click FAB + > New room
4. You need to enter long room name
5. Select any Workspace
6. Tap Create



### Tested On


### Screenshots


#### Web

![image](https://user-images.githubusercontent.com/24370807/151489908-d5f14d83-a80c-4f04-841f-4af7b21e9740.png)

#### Mobile Web

![image](https://user-images.githubusercontent.com/24370807/151489809-730b2bab-c62e-4a7c-9fa4-98333292d0cb.png)

#### Desktop

![image](https://user-images.githubusercontent.com/24370807/151490913-48da28e1-cdc0-41ed-8e5e-bc714abb0e84.png)
![image](https://user-images.githubusercontent.com/24370807/151491056-c54a41ef-d33e-456e-84e5-0da75c26bad2.png)

#### iOS

![image](https://user-images.githubusercontent.com/24370807/151491450-b38253b9-eb72-4c6c-8dd4-a6b6f78cbeb6.png)

#### Android

![image](https://user-images.githubusercontent.com/24370807/151489880-4ba48787-e574-469a-a8f1-9881898b1970.png)
